community/cypher/cypher/src/main/java/org/neo4j/cypher/internal/tracing/TimingCompilationTracer.java

```diff
@@ -117,27 +117,27 @@ final Clock clock()
 
     private static class Query extends Event implements QueryEvent, QueryCompilationEvent
     {
-        private final String query;
+        private final String actualQuery;
```

Comment:
Why?

Reply:
"It's confusing to have a class member with the same name (case differences aside) as its enclosing class. This is particularly so when you consider the common practice of naming a class instance for the class itself."

Reply:
I don't think there was any confusion here. Although, I don't have a strong opinion that it must be `query`, I do however think that `actualQuery` is the wrong name. `queryString` would be better.